aom/src/main/java/com/nedap/archie/rules/ExpressionType.java

```diff
@@ -47,6 +49,8 @@ public String toString() {
                 return "DateTime";
             case DURATION:
                 return "Duration";
+            case C_STRING:
+                return "C_STRING";
```

Comment:
I would guess this should be the name of the type in CamelCase, i.e. `return "CString";`

Reply:
Fixed.